plugins/org.wso2.developerstudio.eclipse.platform.ui/src/org/wso2/developerstudio/eclipse/platform/ui/wizard/AbstractWSO2ProjectCreationWizard.java

```diff
@@ -407,7 +407,10 @@ private IProject getProjectFromModule(String moduleName) {
 	}
 
 	public void createPOM(File pomLocation) throws Exception {
-		MavenInfo mavenInfo = getModel().getMavenInfo();
+		ProjectDataModel modelkushan = getModel();
+		log.info("ProjectDataModel sucssefully created");
+		MavenInfo mavenInfo = modelkushan.getMavenInfo();
+		log.info("MavenInfo sucssefully created");
```

Comment:
1. What is the usefulness of separating in to 2 steps?
2. What is modelkushan???? Who is kushan????? This is absolutely ridiculous!!! CHANGE the variable name
3. Remove INFO Logs. Change to DEBUG